### DISCUSSION TITLE
When is 2.1 support coming? :)

### DISCUSSION THREAD

**Discussion by vdruts** [Mikubill/sd-webui-controlnet#394] [General]
Just curious, does this help?

https://github.com/lllyasviel/ControlNet/commit/2a4424c1c0623d76592d2ef2f486c3de487ef4b5

**Comment by fractal-fumbler** [comment]
https://huggingface.co/alfredplpl/ControlNetForSD2

**Reply by ljleb** [reply]
So if I understand correctly, to work with sd2.x models, control models need to be finetuned specifically for sd2.x?
